backends/graphics/openglsdl/openglsdl-graphics.cpp

```diff
@@ -449,6 +455,12 @@ void OpenGLSdlGraphicsManager::handleResizeImpl(const int width, const int heigh
 	SdlGraphicsManager::handleResizeImpl(width, height);
 }
 
+#ifdef EMSCRIPTEN
+void OpenGLSdlGraphicsManager::saveScreenshot() {
+	_queuedScreenshot = true;
```

Comment:
Would there be a major disadvantage to doing this on all platforms?

Reply:
It adds a delay of 1 frame and generally seems a bit hackish. I would probably keep this for Emscripten only (I'm not aware of any other platform having this issue)

Reply:
Doing this makes sure the screenshot is taken after the whole scene composition is done.
I wonder if we could use a framebuffer as a source for the screenshot instead of taking the next frame.

Reply:
Unfortunately not all engines support rendering to the framebuffer, so I'd suggest to leave this for now.